src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Symbology/UseDistanceCompositeSym/UseDistanceCompositeSym.xaml.cs

```diff
@@ -7,89 +7,99 @@
     {
-        // URL for an image service to use as an elevation source
-        private string _elevationSourceUrl = @"https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
-
         public UseDistanceCompositeSym()
         {
             InitializeComponent();
 
-            // Create the Scene, basemap, graphic, and composite symbol
+            // Create the Scene, basemap, graphic, and composite symbol.
             Initialize();
         }
 
-        private void Initialize()
+        private async Task Initialize()
         {
-            // Create a new Scene with an imagery basemap
+            // Create a new Scene with an imagery basemap.
             Scene myScene = new Scene(Basemap.CreateImagery());
 
-            // Create an elevation source for the Scene
-            ArcGISTiledElevationSource elevationSrc = new ArcGISTiledElevationSource(new Uri(_elevationSourceUrl));
-            myScene.BaseSurface.ElevationSources.Add(elevationSrc);
-
-            // Add the Scene to the SceneView
+            // Add the Scene to the SceneView.
             MySceneView.Scene = myScene;
 
-            // Create a new GraphicsOverlay and add it to the SceneView
+            // Create a new GraphicsOverlay and add it to the SceneView.
             GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
             graphicsOverlay.SceneProperties.SurfacePlacement = SurfacePlacement.Relative;
             MySceneView.GraphicsOverlays.Add(graphicsOverlay);
 
-            // Call a function to create a new distance composite symbol with three ranges
-            DistanceCompositeSceneSymbol compositeSymbol = CreateCompositeSymbol();
+            // Call a function to create a new distance composite symbol with three ranges.
+            DistanceCompositeSceneSymbol compositeSymbol = await CreateCompositeSymbol();
 
-            // Create a new point graphic with the composite symbol, add it to the graphics overlay
+            // Create a new point graphic with the composite symbol, add it to the graphics overlay.
             MapPoint locationPoint = new MapPoint(-2.708471, 56.096575, 5000, SpatialReferences.Wgs84);
             Graphic pointGraphic = new Graphic(locationPoint, compositeSymbol);
             graphicsOverlay.Graphics.Add(pointGraphic);
 
-            // Set the viewpoint with a new camera focused on the graphic
-            Camera newCamera = new Camera(new MapPoint(-2.708471, 56.096575, 5000, SpatialReferences.Wgs84), 1500, 0, 80, 0);
-            MySceneView.SetViewpointCamera(newCamera);
+            // Add an orbit camera controller to lock the camera to the graphic.
+            OrbitGeoElementCameraController cameraController = new OrbitGeoElementCameraController(pointGraphic, 20)
+            {
+                CameraPitchOffset = 80,
+                CameraHeadingOffset = -30
+            };
+            MySceneView.CameraController = cameraController;
         }
```

Comment:
And the end of the try/catch block:
            }
            catch (Exception e)
            {
                MessageBox.Show(e.ToString(), "Error");
            }